### DISCUSSION TITLE
Cannot import django

### DISCUSSION THREAD

**Discussion by Rishabh7777** [microsoft/pylance-release#2308] [Q&A] [answered]
I have installed django in my virtual environment and also selected it in VSCode, but still its giving error with Pylance.
![django error](https://user-images.githubusercontent.com/53748445/151686739-791a2c2a-06b0-41a3-aab3-89b45082f44c.PNG)

**Comment by heejaechang** [comment, marked as answer]
@Rishabh7777 you selected your venv in terminal but didn't select it in vscode. 
![image](https://user-images.githubusercontent.com/1333179/173163018-804bbdf1-f5e4-4a5a-aca7-f76493b3d025.png)
click python in red circle to select right venv for your vscode.
